homeassistant/components/hassio/ingress.py

```diff
@@ -134,7 +134,7 @@ async def _handle_request(
             if (
                 hdrs.CONTENT_LENGTH in result.headers
                 and int(result.headers.get(hdrs.CONTENT_LENGTH, 0)) < 4194000
-            ):
+            ) or result.status == 304:
```

Comment:
I think that we can just approve all 300 responses.
```suggestion
            ) or 300 <= result.status < 400:
```

Reply:
As said 204 should be their to. It's explicity a response without body.

Reply:
```suggestion
            ) or result.status in (204, 304):
```

Reply:
Good point @elupus. Updated suggestion to do 204 too.